Update upgrading-stack.asciidoc (#4567)
elastic/cloud-on-k8s: docs/orchestrating-elastic-stack-applications/upgrading-stack.asciidoc

Follow the instructions in the link:https://www.elastic.co/guide/en/elastic-stack/current/upgrading-elastic-stack.html[Elasticsearch documentation]. Make sure that your cluster is compatible with the target version, take backups, and follow the specific upgrade instructions for each resource type, especially the order in which the upgrade should be carried out. When you are ready, modify the `version` field in the resource spec to the desired stack version and the operator will start the upgrade process automatically.

> Once Elasticsearch is upgraded to the newer version with a green cluster health, you will be able to perform the rest of the stack upgrade. 

Reviewer: ```suggestion
When Elasticsearch is upgraded to the newer version with a green cluster health, you can perform the rest of the stack upgrade. 
```
